components/molecule/drawer/src/index.js

```diff
@@ -0,0 +1,112 @@
+import {useState, useEffect, useRef, forwardRef, useCallback} from 'react'
+import PropTypes from 'prop-types'
+import {createPortal} from 'react-dom'
+import cx from 'classnames'
+
+const Overlay = forwardRef((props, ref) => {
+  return <div ref={ref} className="react-MoleculeDrawer-overlay" {...props} />
+})
+
+const Body = props => {
+  return <div className="react-MoleculeDrawer-body" {...props} />
+}
+
+const Content = forwardRef((props, ref) => {
```

Comment:
For each one of this little components you should add the displayName & propTypes

Reply:
They are just to give semantics, do you think that addind displayName & propTypes will improve anything at all?

Reply:
Hmmmm I don't, and I'm ok this way, BUT, the linter we have mark it as an error if a React Component doesn't have displayName & propTypes. So, up to you 👍

Reply:
If the main purpose is the semantic, i think creating 3 new functions, forwarding ref and adding display names is a bit overengineered, i'd suggest to pass directly the className to the element in the return statement of the component and create renamed tags to give a semantic to the main component:
```
const Body = 'div'
const Overlay = 'div'
const Content = 'div'
```

The pass directly the className where required. I see it was duplicated for the **react-MoleculeDrawer-content**